opendcs-web-client/src/main/webapp/conf/api.conf

```diff
@@ -1,3 +1,3 @@
 url=http://127.0.0.1
-port=8081
-context=odcsapi
\ No newline at end of file
+port=8080
+context=opendcs-rest-api-99.feature-dynamic-sql-SNAPSHOT
```

Comment:
should probably leave this as odcsapi to match the word doc documentation

Reply:
I didn't mean to add this change in.  I will revert that.

Reply:
yeah that makes sense